MainWindow.h

```diff
@@ -114,8 +115,8 @@ class MainWindow : public QMainWindow {
 
   std::unique_ptr<buffers::Project> _project;
   QPointer<RecentFiles> _recentFiles;
-  
-  std::unique_ptr<EventData> _event_data;
+
+  static std::unique_ptr<EventData> _event_data;

```

Comment:
There's already a `static MainWindow::_instance` with a getter somewhere that I added. You can use it to refer to the main window and stuff for now. Please use it as it makes it easier to refactor the static access of the main window later on if we ever need to.